src/test/23-c8ql-queries-stream.ts

```diff
@@ -99,9 +99,9 @@ describe34("C8QL Stream queries", function() {
         .create()
         .then(() => {
           return Promise.all(
-            Array.apply(null, { length: 1000 })
+            Array.apply(null, { length: 1000 } as any)
               .map(Number.call, Number)
-              .map((i: Number) => collection.save({ hallo: i }))
+              .map((i: any) => collection.save({ hallo: i }))
```

Comment:
get rid of any

Reply:
The new Typescript compiler will give us an error. We can @ts-ignore this, leave it as any, or do a refactoring of this part of the code. Do you have any other ideas in mind?